webstruct/text_tokenizers.py

```diff
@@ -109,7 +109,15 @@ def tokenize(self, text):
 class DefaultTokenizer(WordTokenizer):
     def tokenize(self, text):
         tokens = super(DefaultTokenizer, self).tokenize(text)
-        # remove standalone commas and semicolons
+        #remove standalone commas and semicolons
+        #as they broke tag sets, e.g. PERSON->FUNCTION in case "PERSON, FUNCTION"
+
+        #but it has negative consequences, e.g.
+        #etalon:    [PER-B, PER-I, FUNC-B]
+        #predicted: [PER-B, PER-I, PER-I ]
+        #because we removed punctuation
```

Comment:
another nitpick, sorry! Could you please add a whitespace between `#` and the actual comment, to follow pep8? It may also help readability to add a `#` on empty line, as pep8 [says](https://www.python.org/dev/peps/pep-0008/#block-comments)

> Paragraphs inside a block comment are separated by a line containing a single `#`